### PR TITLE
Code clean up by fixing swift lint warnings

### PR DESCRIPTION
**Pull request by kishorepran**
Fix swift lint warnings. Most of warnings have been fixed by the Swift Lint Auto correct tool.

### CODE REVIEW

**Comment by onevcat** [comment]
@kishorepran 

Hi,

Thanks for it. However, currently there is no plan now to introduce a linter tool like this into this project yet. Basically for now I am the only code contributor and a linter won't help much but only make things more complex. If we can get more people involved, it would definitely worth to add a linter or other rules, but maybe not yet for now.

Thank you again for it. These are nice, but just not the time. :)

File: Tests/KingfisherTests/ImageViewExtensionTests.swift [hunk 1]
```diff
@@ -65,31 +64,30 @@ class ImageViewExtensionTests: XCTestCase {
...
             result in
             XCTAssertTrue(progressBlockIsCalled)
             XCTAssertNotNil(result.value)
 
             let value = result.value!
             XCTAssertTrue(value.image.renderEqual(to: testImage))
             XCTAssertTrue(self.imageView.image!.renderEqual(to: testImage))
-            //XCTAssertEqual(self.imageView.kf.taskIdentifier, Source.Identifier.current)
-            
+            // XCTAssertEqual(self.imageView.kf.taskIdentifier, Source.Identifier.current)
```
Comment (thejohnlima) [inline Tests/KingfisherTests/ImageViewExtensionTests.swift:81]: If this line is not required I think it's better removing it

File: Tests/KingfisherTests/KingfisherManagerTests.swift [hunk 2]
```diff
@@ -85,10 +85,14 @@ class KingfisherManagerTests: XCTestCase {
                 XCTAssertNotNil(result.value?.image)
                 XCTAssertEqual(result.value!.cacheType, .none)
                 exp.fulfill()
-        }}}}}
+            }
+        }
+        }
+        }
+        }
```
Comment (thejohnlima) [inline Tests/KingfisherTests/KingfisherManagerTests.swift:92]: What about using Xcode auto indentation?
This way we can have the function `testRetrieveImage`  more organized

```swift
func testRetrieveImage() {
    let exp = expectation(description: #function)
    let url = testURLs[0]

    stub(url, data: testImageData)

    let manager = self.manager!

    manager.retrieveImage(with: url) { result in
        XCTAssertNotNil(result.value?.image)
        XCTAssertEqual(result.value!.cacheType, .none)

        manager.retrieveImage(with: url) { result in
            XCTAssertNotNil(result.value?.image)
            XCTAssertEqual(result.value!.cacheType, .memory)

            manager.cache.clearMemoryCache()

            manager.retrieveImage(with: url) { result in
                XCTAssertNotNil(result.value?.image)
                XCTAssertEqual(result.value!.cacheType, .disk)

                manager.cache.clearMemoryCache()

                manager.cache.clearDiskCache {
                    manager.retrieveImage(with: url) { result in
                        XCTAssertNotNil(result.value?.image)
                        XCTAssertEqual(result.value!.cacheType, .none)
                        exp.fulfill()
                    }
                }
            }
        }
    }

    waitForExpectations(timeout: 3, handler: nil)
}
```

File: Tests/KingfisherTests/KingfisherManagerTests.swift [hunk 3]
```diff
@@ -108,36 +112,40 @@ class KingfisherManagerTests: XCTestCase {
...
                 XCTAssertEqual(result.value!.cacheType, .none)
 
                 exp.fulfill()
-        }}}}}}
+            }
+        }
+        }
+        }
+        }
+        }
```
Comment (thejohnlima) [inline Tests/KingfisherTests/KingfisherManagerTests.swift:133]: Same as my previous [note](https://github.com/onevcat/Kingfisher/pull/1879/files#r762661732)

File: Tests/KingfisherTests/NSButtonExtensionTests.swift [hunk 4]
```diff
@@ -66,20 +66,20 @@ class NSButtonExtensionTests: XCTestCase {
...
             result in
             XCTAssertTrue(progressBlockIsCalled)
-            
+
             let image = result.value?.image
             XCTAssertNotNil(image)
             XCTAssertTrue(image!.renderEqual(to: testImage))
             XCTAssertTrue(self.button.image!.renderEqual(to: testImage))
-            //XCTAssertEqual(self.button.kf.taskIdentifier, Source.Identifier.current)
+            // XCTAssertEqual(self.button.kf.taskIdentifier, Source.Identifier.current)
```
Comment (thejohnlima) [inline Tests/KingfisherTests/NSButtonExtensionTests.swift:80]: It seems this line is a dead code, can we remove it?

File: Tests/KingfisherTests/NSButtonExtensionTests.swift [hunk 5]
```diff
@@ -94,16 +94,15 @@ class NSButtonExtensionTests: XCTestCase {
...
             result in
             XCTAssertTrue(progressBlockIsCalled)
-            
+
             let image = result.value?.image
             XCTAssertNotNil(image)
             XCTAssertTrue(image!.renderEqual(to: testImage))
             XCTAssertTrue(self.button.alternateImage!.renderEqual(to: testImage))
-            //XCTAssertEqual(self.button.kf.alternateTaskIdentifier, Source.Identifier.current)
+            // XCTAssertEqual(self.button.kf.alternateTaskIdentifier, Source.Identifier.current)
```
Comment (thejohnlima) [inline Tests/KingfisherTests/NSButtonExtensionTests.swift:102]: It seems this line is a dead code, can we remove it?

**Comment by onevcat** [comment]
@thejohnlima Yes. These commented code should be removed. Ref: #1883

For the indentation, it is a multiple level nested things and it would be nice if we could rewrite it when async support added (in Xcode 13.2). For now, it is intended to break the format a bit to make it a bit more compact visually and describes a sequence of actions.

File: Tests/KingfisherTests/KingfisherManagerTests.swift (hunk 2 above)
Comment (kishorepran) [reply Tests/KingfisherTests/KingfisherManagerTests.swift:92]: Agree, Xcode indentation would be better.

**Pull request closed without merging**